inexor/fpsgame/client.cpp

```diff
@@ -179,6 +179,7 @@ namespace game
 
     /// change own nick name
     /// @param name my new nickname
+	/// @param tag is clantag of the player
```

Comment:
Nah, thats like the oldest discussion in the world of programming: some people prefer tabs, others spaces..
we agreed on using spaces as we began (but we did not yet reformat everything see #417)

To show whitespaces in vs: https://blogs.msdn.microsoft.com/zainnab/2010/04/09/view-white-space/
and you should enable that vs is always replacing tabs with spaces (thats just more comfortable: https://stackoverflow.com/a/14167067/8043980)